### PR TITLE
Update dynamic-theme-fixes.config

### PR DESCRIPTION
**Pull request by wanzengy**
The image in 'tianchi.aliyun.com' will be unclear under dark mode. So I set image 'no invert'.

### CODE REVIEW

**Comment by Gusted** [comment]
`NO INVERT` is not a option in dynamic theme.
Have a look for the avaible commands in Dynamic theme
https://github.com/darkreader/darkreader/blob/master/README.md#use-dev-tools

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -6011,3 +6018,4 @@ zhihu.com
 
 INVERT
 img[eeimg="1"]
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:6021]: You can remove this new line.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -4733,6 +4733,13 @@ INVERT
 
 ================================
 
+tianchi.aliyun.com
+
+NO INVERT
+img
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4739]: `img` This is every image, try to just select certain images who need it.

Comment (wanzengy) [reply src/config/dynamic-theme-fixes.config:4739]: Thank you for your reply. And actually all `image`s showed in `tianchi.aliyun.com` have some problems, for example the words in image can't see clearly under dark mode. So I set all images.

**Pull request closed without merging**